services-interfaces/src/main/java/io/kaoto/backend/api/service/step/parser/StepParserService.java

```diff
@@ -40,6 +40,16 @@
     @WithSpan
     ParseResult<T> deepParse(String yaml);
 
+    /*
+     * 🐱method getParsedFlows: List<ParseResult>
+     * 🐱param yaml: String
+     *
+     * Based on the YAML provided, offer a list of flows defined on it and
+     * the metadata associated to the orchestration.
+     */
+    @WithSpan
+    List<ParseResult<T>> getParsedFlows(String yaml);
```

Comment:
This is a new method to return multiple flows from a CRD, but I'm not sure if this is the right approach.

Reply:
I guess it makes sense. We will deprecate the single flow return at some point.